CHMeetupApp/Sources/Common/Extensions/Foundation/String/String+Range.swift

```diff
@@ -10,12 +10,12 @@ import Foundation
 
 extension String {
   func substring(from index: Int) -> String {
-    return substring(from: self.index(startIndex, offsetBy: index))
+    return String(self[self.index(startIndex, offsetBy: index)...])
```

Comment:
let ind = self.index(startIndex, offsetBy: index)
return String(self[ind...])

easier to read 👆

Reply:
```
return String(self[index(startIndex, offsetBy: index)...])
```

Reply:
No)) index here is local, but I need self.

Reply:
You are right. Sorry.